src/main/deltachat.js

```diff
@@ -6,24 +6,22 @@ const config = require('../config')
 
 class ChatMessage {
   constructor (messageId, dc) {
-    var msg = dc.getMessage(messageId)
-    this.fromId = msg && msg.getFromId()
-    if (!this.fromId) return
```

Comment:
I removed this one and made something similar in `toJson()`. I think the previous commit would have crashed on `this.contact.toJson()` since `this.contact` would have been `undefined` in case `fromId` was falsey (since we return in the constructor and never set `this.contact`).